packer/scripts/ubuntu/resolv.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+echo $dnsserver1  >> /etc/resolvconf/resolv.conf.d/base
+echo $dnsserver2 >> /etc/resolvconf/resolv.conf.d/base
```

Comment:
This won't actually pull the var from the packer template like you want. Take a look at how ``CHEF_VERSION`` is set and used in the chef script.